litmus-portal/frontend/public/locales/en/translation.yaml

```diff
@@ -198,9 +287,150 @@ settings:
         title: Change your Avatar
         info: You can now use your new password to login to your account
         button: Choose an avatar
-    deleteUser:
-      deleteModal:
-        body: The user will lose access to the portal
+      newUserModal:
+        header: A new user <1>{{name}}</1> was successfully created
+        headerErr: <0> Error </0> while creating a new user.
```

Comment:
I've always seen `<b>` tags for bold text, can you please explain these `<1>` and `<0>` tags? Are they specific to some API?

Reply:
Yes, they are specific to the `Trans` API. It allows us to integrate React nodes in our translated text. So if we have a formatted piece of text in our jsx like `<strong>Error</strong> while creating a new user`, we can use `Trans` to translate that easily by specifying that in the yaml using replacement tags(`<0>`/`<1>`) which are written with respect to the position of the React node, starting from 0. So if there is a formatting tag as the first node it'll be replaced by `<0>`, if it is in the second node we'll need to use `<1>` and so on.

Reply:
I guess this would change if we decide not to go with the <Trans /> approach